src/keycloak/keycloak-ees-realm.json

```diff
@@ -492,7 +492,7 @@
     "emailVerified" : true,
     "firstName" : "Prerelease1",
     "lastName" : "User1",
-    "email" : "ees-prerelease1@education.gov.uk",
+    "email" : "ees-test.prerelease1@education.gov.uk",
```

Comment:
At the moment we haven't migrated to DFE-based test pre-release users yet. I believe any pre-release users can stay the same (in order to avoid failures in the UI tests)

Reply:
Done, ta!